index.html

```diff
@@ -0,0 +1,231 @@
+                        </svg>
+                   </a>
+               </li>
+               </ul>
+            </div>
+        </section>
+
+        <!--Dynamic rendering of cards in card container-->
+        <div class="cards">
+            <div class="card 1">  
+                <img src="imgs/poster.svg" alt="Snapshoot-Portfolio">
+                    <div class="card-block" id="arti-1">
+                        <h2>Tonic</h2>
+                        <ul class="card-frame">
+                            <li class="no-count">CANOPY</li>
+                            <li>Back End Dev</li>
+                            <li>2015</li>
+                        </ul>
```

Comment:
- [ ] Kindly add the dots as requested by the previous reviewer 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/205448882-c1624a94-171c-4ef0-877b-fb110ebcc7a7.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/205448893-cb8abba6-fd51-430e-823c-e4cc8bc69022.png)